packages/x-scheduler/src/joy/event-calendar/EventCalendar.tsx

```diff
@@ -13,12 +14,12 @@ export const EventCalendar = React.forwardRef(function EventCalendar(
 
   return (
     <div
-      className={clsx(className, 'EventCalendarRoot', 'mui', 'light')}
+      className={clsx(className, 'EventCalendarRoot', 'mui', 'light', 'joy')}
```

Comment:
I think you can remove the "mui" class and move its logic to the "joy" class
I don't think we should have any trace of "mui" in the new component :ok_hand: